docs/nodes/bridge-node.mdx

```diff
@@ -51,7 +51,7 @@ From an implementation perspective, Bridge Nodes run two separate processes:
 The following hardware minimum requirements are recommended for running the
 bridge node:
 
-* Memory: **8 GB RAM**
+* Memory: **24 GB RAM**
 * CPU: **6 cores**
 * Disk: **1 TB SSD Storage**
```

Comment:
I think we should also recommend a different value here. 

Cc @walldiss, @Bidon15

Reply:
ack. will wait on your decision here